src/status_im/keycard/real_keycard.cljs

```diff
@@ -107,6 +107,17 @@
                 (on-success info))))
       (catch on-failure)))
 
+(defn factory-reset
+  [{:keys [on-success on-failure]}]
+  (.. status-keycard
+      (factoryReset)
+      (then (fn [response]
+              (let [info (-> response
+                             (js->clj :keywordize-keys true)
+                             (update :key-uid ethereum/normalized-hex))]
+                (on-success info))))
+      (catch on-failure)))
```

Comment:
Does `catch` works without a `try` block?

Reply:
Yes the same pattern is used throughout the file and the error handlers are known to work. I would leave it like this for consistency. Thanks for the review!